*****************************************************************************************************************
*   FILENAME    *    PRIORITY    *  TEST-CASE LENGTH  *  NUMBER OF ACTIONS  *    BUG PRODUCED  *  EXCEPTION
*****************************************************************************************************************
* Readme  *      HIGH      *         4         *       0         *       True     *  test-case failed to execute with following message: b'  File "Readme", line 1\r\n    This empty-file is needed to run AGTGAdriver.press_keycode(3) \r\n             ^\r\nSyntaxError: invalid syntax\r\n'
_________________________________________________________________________________________________________________
* TestCase00.py  *       LOW      *         19         *       3         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase01.py  *       LOW      *         25         *       5         *       False     *  Server-side error occurred. Please rerun this test-case
_________________________________________________________________________________________________________________
* TestCase02.py  *       LOW      *         19         *       3         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase03.py  *       LOW      *         16         *       2         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase04.py  *       LOW      *         19         *       3         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase05.py  *       LOW      *         34         *       8         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase06.py  *       LOW      *         19         *       3         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase07.py  *       LOW      *         28         *       6         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase08.py  *       LOW      *         19         *       3         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase09.py  *       LOW      *         25         *       5         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase10.py  *       LOW      *         19         *       3         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase11.py  *      HIGH      *         37         *       9         *       True     *  test-case failed to execute with following message: b'Traceback (most recent call last):\r\n  File "TestCase11.py", line 28, in <module>\r\n    el.click()\r\nAttributeError: \'NoneType\' object has no attribute \'click\'\r\n'
_________________________________________________________________________________________________________________
* TestCase12.py  *       LOW      *         16         *       2         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase13.py  *       LOW      *         28         *       6         *       False     *  Server-side error occurred. Please rerun this test-case
_________________________________________________________________________________________________________________
* TestCase14.py  *       LOW      *         25         *       5         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase15.py  *       LOW      *         31         *       7         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase16.py  *       LOW      *         22         *       4         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase17.py  *       LOW      *         19         *       3         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase18.py  *       LOW      *         25         *       5         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase19.py  *       LOW      *         19         *       3         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase20.py  *       LOW      *         22         *       4         *       False     *  None
_________________________________________________________________________________________________________________
* TestCase21.py  *       LOW      *         10         *       0         *       False     *  None
_________________________________________________________________________________________________________________
*   Average Test-Case Length               =                         23.894736842105264
*   Average Action per Test-case           =                         4.631578947368421
*****************************************************************************************************************

Process finished with exit code 0
